pkgs/applications/editors/vim/plugins/aliases.nix

```diff
@@ -45,6 +45,8 @@ mapAliases (with prev; {
   coffeeScript        = vim-coffee-script; # backwards compat, added 2014-10-18
   Solarized           = vim-colors-solarized;
   solarized           = vim-colors-solarized;
+  spacevim            = throw "this distribution was unmaintained for the last 6 years, please use top-level 'spacevim'"; # added 2024-11-27
+  SpaceVim            = throw "this distribution didn't work properly in vimPlugins, please use top-level 'spacevim' instead"; # added 2024-11-27
```

Comment:
Assuming you actually tried using it and that's why you updated the top level one?

Reply:
Yes, top-level one now works

![image](https://github.com/user-attachments/assets/ca8adff1-99d6-4daf-84ef-dca688c6febf)